lib/msal-browser/src/app/PublicClientApplication.ts

```diff
@@ -85,6 +90,36 @@ export class PublicClientApplication extends ClientApplication implements IPubli
         if (!account) {
             throw BrowserAuthError.createNoAccountError();
         }
+        const thumbprint: RequestThumbprint = {
+            clientId: this.config.auth.clientId,
+            authority: request.authority || "",
+            scopes: request.scopes,
+            homeAccountIdentifier: account.homeAccountId
+        };
+        const silentRequestKey = ThrottlingUtils.generateThrottlingStorageKey(thumbprint);
```

Comment:
Can we rename and/or move this function if this is going to start being used for things other than throttling?

Reply:
@tnorling Would changing the name to `generateRequestStorageKey` be enough, or should it also be moved out of `ThrottlingUtils`? Aside from here, the only other place it is used is in `ThrottlingUtils`.

Reply:
`generateThrottlingStorageKey` just does this: ``${ThrottlingConstants.THROTTLING_PREFIX}.${JSON.stringify(thumbprint)}`;`

Since we don't need the throttling prefix here, instead of moving/renaming can we just do: 
```suggestion
        const silentRequestKey = JSON.stringify(thumbprint);
```

Reply:
Makes sense. Changed.